plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx

```diff
@@ -54,6 +59,8 @@ function getInterpolatedSizeLabel( mixedString: string ) {
 export const ProductShippingSection: React.FC = () => {
 	const { getInputProps } = useFormContext< Product >();
 	const { formatNumber, parseNumber } = useProductHelper();
+	const [ highlightSide, setHighLightSide ] =
```

Comment:
Nitpicky, but should be `setHighlightSide`. ("Highlight" being one word)

Reply:
Absolutly ;)